modules/secured-cluster-services-config.adoc

```diff
@@ -212,6 +212,63 @@ Otherwise, you must ensure that Central can access the online probe repository h
 | `auditLogs.disableCollection`
 | If you set this option to `true`, {product-title} disables the audit log detection features used to detect access and modifications to configuration maps and secrets.
 
+| `scanner.disable`
+| If you set this option to `true`, {product-title} deploys a lightweight scanner and scannerdb in the secured cluster to allow scanning images on OpenShift Container Registry. It's reccomended this be set to true on OpenShift or clusters relying on a registry not exposed outside of the cluster. Defaults to `false`
```

Comment:
a few things on this line:

* "If you set this option to `true`" -> "If you set this option to `false`".
* nit: `reccomended` -> `recommended`
* Let's just say OpenShift for now since we really only officially support the OpenShift Internal Image Registry and not other cluster-internal registries.
* I'm not sure the naming convention used, but does "secured cluster" implied Central is not present in this cluster? Because the lightweight Scanner and Scanner DB will only exist in non-Central clusters. The cluster with Central will still have the normal Scanner and Scanner DB

Reply:
Also, is it true it defaults to `false`? I'm actually not sure @juanrh @porridge ?

Reply:
Good catch @RTann I mixed them up. Even looked at the code but wrote it backwards.

Reply:
Fixed. Reviewed the code and its also only supported on OpenShift so modified this. Actually errors out when the environment isn't openshift.